src/app.js

```diff
@@ -1,29 +1,26 @@
-//Importamos configuracion de Express
-import app from './config/express.js';
-
-//Importamos rutas
-import routes from './routes/index.route.js';
-
-import mongoose from 'mongoose';
-
+//Para usar require, hay que importar esto
```

Comment:
I strongly recommend you not to add this kind of comment (some `TODO` or something strictly necessary, it's ok, but not in general). Your code should be self-explanatory, and if it's not, then you should change something probably.